addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/CrossDomainScriptInclusionScanRule.java

```diff
@@ -84,17 +84,28 @@ && isScriptFromOtherDomain(
         }
     }
 
-    private void raiseAlert(HttpMessage msg, int id, String crossDomainScript, String evidence) {
-        newAlert()
+    private AlertBuilder createAlert(String crossDomainScript, String evidence) {
+        return newAlert()
                 .setRisk(getRisk())
                 .setConfidence(Alert.CONFIDENCE_MEDIUM)
                 .setDescription(getDescription())
                 .setParam(crossDomainScript)
                 .setSolution(getSolution())
                 .setEvidence(evidence)
                 .setCweId(getCweId())
-                .setWascId(getWascId())
-                .raise();
+                .setWascId(getWascId());
+    }
+
+    private void raiseAlert(HttpMessage msg, int id, String crossDomainScript, String evidence) {
+        createAlert(crossDomainScript, evidence).raise();
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(createAlert(
+                "<script type=\"text/javascript\" src=\"externalDomain.example.com/weatherwidget.js\"></script>",
+                null
```

Comment:
I believe the parameters are swapped.

Reply:
You are right, thank you! The `evidence` variable is the whole script tag and the `crossDomainScript` is just the `src` attribute, right?